README.md

```diff
@@ -59,8 +61,9 @@ func main() {
 
 	c.SubscribeTopics([]string{"myTopic", "^aRegex.*[Tt]opic"}, nil)
 
-	for {
-		msg, err := c.ReadMessage(-1)
+	run := true // A signal handler or similar could be used to set this to false to break the loop.
+	for run {
+		msg, err := c.ReadMessage(5 * time.Second)
```

Comment:
If using a timeout you will now also need to handle the timeout case specifically below, since it is not an error.

Reply:
Thank @edenhill, I wasn't sure what would be the best way for this snippet.

I added the timeout because without the timeout, the loop never terminates if we are polling to a topic that messages aren't being produced to. 

I can remove it entirely and keep it -1, if that is what you suggest to keep the example simple.

Reply:
No I think it makes sense to have a timeout here as the run var would be a bit useless otherwise.
But you need to check for the timeout case, I think it is the msg==nil,err==nil, but please double check (and test).

Reply:
I think with ReadMessage we give an `err.(kafka.Error).Code() != kafka.ErrTimedOut`, I'll confirm and change it